docs/pages/Configuration.md

```diff
@@ -465,3 +465,11 @@ Version 1.0 projects are supported by [UI5 CLI](https://github.com/SAP/ui5-cli)
 Initial version.
 
 Version 0.1 projects are compatible with [UI5 CLI](https://github.com/SAP/ui5-cli) v0.0.1 and above.
+
+## Validation / IDE support
```

Comment:
I would rather move this chapter to the very beginning of this file. Before the "Example".

Reply:
I agree - the info in Spec Versions is fundamental when working on configurations, hence should come first.

Reply:
Done